engine.py

```diff
@@ -1693,18 +1694,15 @@ def _jump_to_fs(self):
         self.logger.debug("FS paths: %s" % (str(paths),))
         for disk_location in paths:
 
-            # get the setting
-            system = sys.platform
-
             # run the app
-            if system == "linux2":
-                cmd = 'xdg-open "%s"' % disk_location
-            elif system == "darwin":
+            if sgtk.util.is_macos():
```

Comment:
I've seen this code in soooo many places, we should definitely refactor it... Not now. But someday.